autosklearn/estimators.py

```diff
@@ -263,7 +263,7 @@ def __init__(
         self.logging_config = logging_config
         self.metadata_directory = metadata_directory
 
-        self._automl = None  # type: Optional[List[BaseAutoML]]
```

Comment:
Hm, what's the reason to drop this? Was it incorrect?

Reply:
I removed the comment here. The logic behind this was:

There was an unnecessary import here to BaseAutoML. I thought it was a legacy due to the comment:
self._automl = None  # type: Optional[List[BaseAutoML]

Here, the automl attribute defaults to None. The comment doesn't make sense? Please let me know if you still want to keep that comment!

Reply:
Hey, that's a type hint: https://docs.python.org/3/library/typing.html

We don't use them as much as we should yet, but I'd like to keep it.